packages/project-access/src/project/search.ts

```diff
@@ -1,4 +1,5 @@
 import { basename, dirname, isAbsolute, join, parse, sep } from 'path';
+import path from 'path';
```

Comment:
Why do you import `path` here? this leads to a [codesmell](https://sonarcloud.io/project/issues?id=SAP_open-ux-tools&pullRequest=2438&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true) and `sep` is already imported. Then you could also avoid renaming the path varible.

Reply:
Thank you, I have updated the code.